### PR TITLE
Added broad landcover data and README explaining the data retrieval

### PR DESCRIPTION
**Pull request by egrace479**
Updated landcover information for broad regions. Includes files for generation of the landcover information and a README for explanation and citations.

### CODE REVIEW

File: datasets/data_retrieval/mapping_broad.py [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+                                    length_unit = 9001, # meters
+                                    area_unit = 9001, 
+                                    calculation_type = 'preserveShape', 
+                                    spatial_ref = aoi_spatial_ref)
+###
+    # area = arcgis.geometry.areas_and_lengths(polygons = intersection[0], 
+    #                                          length_unit= 9001, 
+    #                                          area_unit=9001, 
+    #                                          calculation_type='geodesic', 
+    #                                          spatial_ref = spatial_ref)
```
Comment (johnbradley) [inline datasets/data_retrieval/mapping_broad.py:255]: Stray bit of commented code you may want to remove.

File: datasets/data_retrieval/mapping_broad.py [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+    aoi - Area of interest. ArcGIS Polygon object.
+    feature_areas - DataFrame containing regions and the area of their intersection with the given area of interest (lat/lon).
+
+    Returns:
+    --------
+    region_percents - Dictionary of percentage of land in area of interest covered by each region category (index: Grassy, Dense Wood, Woody, Suburban, Watery, Urban).
+
+    '''
+    #aoi = get_area_of_interest(lat, lon, radius)
+    #feature_areas = get_aoi_feature_areas(layer, aoi)
```
Comment (johnbradley) [inline datasets/data_retrieval/mapping_broad.py:286]: Another piece of commented out code.

**Review by johnbradley — APPROVED** [review]
LGTM

File: datasets/data_retrieval/mapping_broad.py (hunk 2 above)
Comment (egrace479) [reply datasets/data_retrieval/mapping_broad.py]: Removed, and adjusted function parameter description to reflect the removal of these lines.

**Pull request merged**